internal/sdkconstants/byte.go

```diff
@@ -0,0 +1,12 @@
+package sdkconstants
```

Comment:
would it make sense to put this kind of constants in the [sdkio](https://github.com/aws/aws-sdk-go/tree/master/internal/sdkio) package, instead of creating a new package?